src/current/v23.2/set-up-physical-cluster-replication.md

```diff
@@ -0,0 +1,313 @@
+
+## Step 3. Verify certificates
+
+At this point, you have a primary and a standby cluster running. The next step is to verify that the clusters can communicate. Depending on how you manage certificates, you must ensure that all nodes on the primary and the standby cluster have access to the certificate of the other cluster.
+
+For example, if you followed the [Deploy CockroachDB]({% link {{ page.version.version }}/deploy-cockroachdb-on-premises.md %}) prerequisite, you need to add the `ca.crt` from the primary cluster to the `certs` directory on all the nodes in the standby cluster.
+
+1. Name the `ca.crt` from the primary cluster to a new name on the standby cluster. For example, `ca_primary.crt`.
+1. Place `ca_primary.crt` in the `certs` directory of the standby cluster nodes.
+
+You need to add the `ca.crt` from the standby cluster to the `certs` directory on all the nodes in the primary cluster.
+
+1. Name the `ca.crt` from the standby cluster to a new name on the primary cluster. For example, `ca_standby.crt`.
+1. Place `ca_standby.crt` in the `certs` directory of the primary cluster nodes.
+
+## Step 4. Start replication
+
+The system interface in the standby cluster initiates and controls the replication stream by pulling from the primary cluster. In this section, you will connect to the primary from the standby to initiate the replication stream.
+
+1. From the **standby** cluster, use your connection string to the primary.
+
+    The connection string contains:
+    - The replication user and password that you [created for the primary cluster](#create-a-replication-user-and-password).
+    - The node IP address or hostname of one node from the primary cluster.
+    - The path to the primary node's certificate on the standby cluster.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE VIRTUAL CLUSTER standbyapplication LIKE template
+    FROM REPLICATION OF application
+    ON 'postgresql://{replication user}:{password}@{node IP or hostname}:26257/?options=-ccluster=system&sslmode=verify-full&sslrootcert=certs/{primary cert}.crt';
+    ~~~
+
+    {% include {{ page.version.version }}/physical-replication/like-description.md %}
+
+    Once the standby cluster has made a connection to the primary cluster, the standby will pull the topology of the primary cluster and will distribute the replication work across all nodes in the primary and standby.
+
+1. To manage the replication stream, you can pause and resume the replication stream as well as show the current details for the job:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    ALTER VIRTUAL CLUSTER standbyapplication PAUSE REPLICATION;
+    ~~~
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    ALTER VIRTUAL CLUSTER standbyapplication RESUME REPLICATION;
+    ~~~
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SHOW VIRTUAL CLUSTER standbyapplication WITH REPLICATION STATUS;
```

Comment:
It might be worth showing the concise output from `SHOW VIRTUAL CLUSTERS;` here and point out the `replicating` status. I found it easier to read.

Reply:
Added this right beneath the replication stream creation command.